docusaurus/docs/quickstarts/network/local-with-docker.md

```diff
@@ -85,20 +85,39 @@ docker run \
   /openziti/scripts/run-controller.sh
 ```
 
+## Create Edge Router Policies
+OpenZiti requires specific authorization of users and routers (edge router policies), as well as authorization of 
+services and routers (service edge router policies). The docker-based quickstart doesn't perform these steps 
+automatically. Run the initialization container one time, after starting the controller as shown

```

Comment:
felt just a tiny bit unfinished to me? maybe just tag on the end "run as shown, to setup the default authorization"?